functions/sum.py

```diff
@@ -1,51 +1,59 @@
-import helpers.riot_helper as riot_helper
```

Comment:
I can't remember why `sum.py` is even being edited here. But just revert this back to `main`